### ISSUE TITLE
convert ONNX outputs to Bbox

### ISSUE THREAD

**Issue by chen2mg** [ultralytics/yolov5#8085]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and found no similar feature requests.

### Description

I converted my customized "yolo.pt" model to "yolo.onnx".

The output dimensions are:
![Screen Shot 2022-06-02 at 10 40 23 PM](https://user-images.githubusercontent.com/16389955/171776709-7ba0faeb-5f9c-4191-95d2-8382365ffb72.png)

I have read https://github.com/ultralytics/yolov5/issues/708

people suggested "non_max_suppression", but I still have trouble to feed outputs from onnx to "non_max_suppression".

can some one be more specific?

how to convert:
[1, 25200, 7]
[1,3,80, 80,7]
[1,3,40, 40,7]
[1,3,20, 20,7]
to Bbox, labels?

Thank you so much!

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

**Comment by AyushExel** [comment]
@chen2mg You can use detect.py to run inference on exported models directly.
`python path/to/detect.py --yolov5s.onnx   # ONNX Runtime or OpenCV DNN with --dnn`

**Comment by glenn-jocher** [comment]
@chen2mg your code is out of date. To update:
- **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
- **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
- **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  <a href="https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolov5"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image <a href="https://hub.docker.com/r/ultralytics/yolov5"><img src="https://img.shields.io/docker/pulls/ultralytics/yolov5?logo=docker" alt="Docker Pulls"></a>

**Comment by chen2mg** [comment]
> @chen2mg You can use detect.py to run inference on exported models directly. `python path/to/detect.py --yolov5s.onnx # ONNX Runtime or OpenCV DNN with --dnn`

Thank you for your reply.
I also need to deploy this onnx model on Javascript, so I need to know how to convert [1, 25200, 7] to Bbox.

**Comment by chen2mg** [comment]
> @chen2mg your code is out of date. To update:
> 
> * **[Git](https://github.com/ultralytics/yolov5)** – `git pull` from within your `yolov5/` directory or `git clone https://github.com/ultralytics/yolov5` again
> * **[PyTorch Hub](https://pytorch.org/hub/ultralytics_yolov5/)** – Force-reload `model = torch.hub.load('ultralytics/yolov5', 'yolov5s', force_reload=True)`
> * **[Notebooks](https://github.com/ultralytics/yolov5/blob/master/tutorial.ipynb)** – View updated notebooks  [![Open In Colab](https://camo.githubusercontent.com/84f0493939e0c4de4e6dbe113251b4bfb5353e57134ffd9fcab6b8714514d4d1/68747470733a2f2f636f6c61622e72657365617263682e676f6f676c652e636f6d2f6173736574732f636f6c61622d62616467652e737667)](https://colab.research.google.com/github/ultralytics/yolov5/blob/master/tutorial.ipynb) [![Open In Kaggle](https://camo.githubusercontent.com/a08ca511178e691ace596a95d334f73cf4ce06e83a5c4a5169b8bb68cac27bef/68747470733a2f2f6b6167676c652e636f6d2f7374617469632f696d616765732f6f70656e2d696e2d6b6167676c652e737667)](https://www.kaggle.com/models/ultralytics/yolov5)
> * **[Docker](https://hub.docker.com/r/ultralytics/yolov5)** – `sudo docker pull ultralytics/yolov5:latest` to update your image [![Docker Pulls](https://camo.githubusercontent.com/280faedaf431e4c0c24fdb30ec00a66d627404e5c4c498210d3f014dd58c2c7e/68747470733a2f2f696d672e736869656c64732e696f2f646f636b65722f70756c6c732f756c7472616c79746963732f796f6c6f76353f6c6f676f3d646f636b6572)](https://hub.docker.com/r/ultralytics/yolov5)

Thank you for your reply. What do you mean? the export.py I used is out of date?

**Comment by chen2mg** [comment]
problem solve by:

```
# outputs[0]: (1, 25200, 7)
y = torch.tensor(outputs[0]) if isinstance(outputs[0], np.ndarray) else outputs[0]
pred = non_max_suppression(y)
print(pred)

#[tensor([[163.18411, 270.04858, 350.99985, 399.60071,   0.93833,   1.00000],
#        [ 62.01051, 235.79718, 543.79761, 432.11530,   0.93351,   0.00000]])]
```

Thanks

**Comment by glenn-jocher** [comment]
@chen2mg glad to hear that you were able to solve the problem! If you have any more questions or need further assistance, feel free to ask. 😊
